apps/laboratory/src/pages/library/solana.tsx

```diff
@@ -22,7 +29,14 @@ const modal = createWeb3Modal({
   enableAnalytics: false,
   termsConditionsUrl: 'https://walletconnect.com/terms',
   privacyPolicyUrl: 'https://walletconnect.com/privacy',
-  customWallets: ConstantsUtil.CustomWallets
+  customWallets: ConstantsUtil.CustomWallets,
+  adapters: [
```

Comment:
Would it be possible to change the naming here from `adapters` to `wallets`?
We will introduce an adapter flag for chains and might collide with this one.